common/main/en.xml

```diff
@@ -426,6 +426,7 @@ annotations.
 			<language type="mua">Mundang</language>
 			<language type="mul">Multiple languages</language>
 			<language type="mus">Muscogee</language>
+			<language type="mus" alt="variant">Mvskoke</language>
```

Comment:
```suggestion
			<language type="mus" alt="variant">Muscogee</language>
			<language type="mus" alt="official">Mvskoke</language>
```

Reply:
See the ticket comment from the TC. It needs to have 2 alt variants.
https://unicode-org.atlassian.net/browse/CLDR-17761?focusedCommentId=176091

Reply:
That approach was discussed in the TC but there is currently no other language using the tag `alt="official"` -- only "menu" and "variant". Is there documentation mentioning "official" as a supported value for the tag `alt`?

Reply:
Or said differently:  I just didn't see any previous language referring to alt="official" although I can definitely add it if its a supported value.

Reply:
If we make that change,  I can update the official documentation for the field https://cldr.unicode.org/translation/displaynames/languagelocale-names